templates/Search/form.html.twig

```diff
@@ -1,69 +1,104 @@
-                            selected
-                            {% endif %}>
-                        Abstract
-                    </option>
-                    <option
-                            value={{ constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_AUTHORS') }}
-                             {% if field is defined and field is not empty and field == constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_AUTHORS')%}
-                            selected
-                            {% endif %}>
-                        Author
-                    </option>
-                    <option
-                            value={{ constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_THEME_KEYWORDS') }}
-                             {% if field is defined and field is not empty and field == constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_THEME_KEYWORDS')%}
-                            selected
-                            {% endif %}>
-                        Theme Keywords
-                    </option>
-            </select>
-        </div>
-        <div class="input-group col-md-4">
-            <input type="search" id="searchBox" name="query" class="search-query form-control" placeholder="Search..."
-                   value = "{{ query|default }}">
-            {% if page is defined and page is not null%}
-                <input type="hidden" id="pageNo" name="page" value="{{ page | default }}">
-            {% endif %}
-            <span class="input-group-btn">
-                <button id="searchSubmit" class="btn btn-danger" type="submit">
-                    <span class=" glyphicon glyphicon-search"></span>
-                </button>
-            </span>
+<!-- ========================= SECTION CONTENT ========================= -->
+<section class="section-content bg pt-5">
+    <div class="container">
+        <div class="card card-header">
+            <form id="searchForm" name="searchForm" action="{{ path('pelagos_app_ui_searchpage_default') }}" method="get">
+                <div class="row">
+                    <div class="col-sm-9">
+                        <input type="search"
+                               name="query"
+                               class="form-control"
+                               placeholder="Search.."
+                               id="searchBox"
+                               value = "{{ query|default }}"
+                               required>
+                        {% if page is defined and page is not null%}
+                            <input type="hidden" id="pageNo" name="page" value="{{ page | default }}">
+                        {% endif %}
+                    </div>
+                    <div class="col-sm-3 btn-toolbar">
+                        <button id="searchSubmit" type="submit" class="btn btn-primary mx-2 w-50">Search
+                            <i class="fa fa-search"></i></button>
+                        <button type="reset" class="btn btn-dark mx-2 w-25">Clear</button>
```

Comment:
This button doesn't work?

Reply:
It does a basic HTML reset, I'm not sure what would the PO's want for that button to do